collections/file_systems/roles/xfs_quota/readme.rst

```diff
@@ -0,0 +1,66 @@
+Xfs_quota
+----------
+
+Description
+^^^^^^^^^^^
+
+Used to set default user and group disk quotas on xfs disk used by server.
+
+
+**There are 3 types of quotas:**
+
+- user
+- group
+- *project Not implemented*
+
+
+
+
+This role provides simply provides an interface to `**xfs_quota** Ansible module <https://docs.ansible.com/ansible/latest/collections/community/general/xfs_quota_module.html>`_
+
+**Observation**
+^^^^^^^^^^^^^^^
+
+For this role to work, the filesystem mount configurations must have the options uquota, gquota, pquota .
+
+Example 
+
+/dev/xvdb1 /xfs xfs rw,quota,gquota,pquota 0 0
+
+
+Instructions
+^^^^^^^^^^^^
+Set mountpoint for user and group default values in quota_filesystem
+
+Set specific values for user and groups outside the default scope in quota_spec
+
+
+
+.. code-block:: yaml
+
+   quota_filesystem:
+    -name: FS1
+     mountpoint: XXXXXX
+
+    -name: FS2
+     mountpoint: XXXXXX
+
+   quotas_spec:
+    - type: user
+      name: nobody 
+      mountpoint: /exports/nfs
+      soft: 5G
+      hard: 6G
+    
+    - type: group
+      name: nobody
+      mountpoint: /exports/nfs
+      soft: 5G
+      hard: 6G
+
+
+See `**xfs_quota** Ansible module page <https://docs.ansible.com/ansible/latest/collections/community/general/xfs_quota_module.html>`_ for the full list of available parameters.
```

Comment:
Beware, you refer to the module for all parameters available but in your code not all of them are added. You should add all of them, like the other wrapper roles.

Reply:
But in this role am I following the scheme for bsoft and bhard values need to add referring to inodes?